### PR TITLE
Create bower.json

### PR DESCRIPTION
**Pull request by ghost**
For use with Bower.io, ie. via http://rails-assets.org/ which would eliminate the need of repos such as https://github.com/ecleel/jquery-oembed-rails.

### CODE REVIEW

**Comment by potench** [comment]
Thanks for the PR @hakkiplaten.  Sign the CLA above and we'll get this landed.

**Comment by hakkiplaten** [comment]
All done. Thanks!

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "homepage": "https://github.com/nfl/jquery-oembed-all",
+  "authors": [
+    "National Football League <engineers@nfl.com>"
+  ],
+  "description": "Use native oEmbed services when possible.",
+  "main": "jquery.oembed.js",
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
```
Comment (potench) [inline bower.json]: I think this needs a couple Updates 

- `jquery` should be included as a `dependency`
- `node_modules`, `bower_components` and  `test(s)` don't need to be ignored cause they don't exist in this module. You should instead ignore `examples/`, `*.md`, and `CHANGELOG`.

**Comment by hakkiplaten** [comment]
Good catch! Changes committed. jQuery should already be included as a dependency, however I'm not sure about the version number.

**Comment by hakkiplaten** [comment]
Had an invalid version number. Should be fixed now.

**Comment by hakkiplaten** [comment]
Could we merge https://github.com/nfl/jquery-oembed-all/pull/41? Bower should automatically pick up on it and add it to its available versions, which in turn should fix `jquery-oembed-all has no versions defined. Please create an issue in component's repository.` at https://rails-assets.org/components/new.

Thank you!

**Comment by ghost** [comment]
Turns out this field is now ignored by Bower:

https://github.com/bower/spec/blob/master/json.md#version

Courtesy of @hut8:

https://github.com/tenex/rails-assets/issues/82

@potench hence, would you mind merging the latest and renaming the git tag to a semver one instead (ie. `0.9.0-rc1`)?

Ref. https://github.com/npm/node-semver#range-grammar

**Comment by potench** [comment]
@hakkiplaten let's remove the bower config. Not sure why anyone would use bower over npm at this point anyways.

**Comment by ghost** [comment]
@potench this is discussed at length at https://github.com/rails-assets/rails-assets/issues/291, but the common consensus seems to be that Bower and the Bower proxy [Rails-Assets.org](http://rails-assets.org/) is currently the simplest way for Ruby on Rails developers to maintain their JS dependencies.

JS front-end development is moving too fast, and current NPM solutions for Ruby on Rails are too complex.

**Comment by hut8** [comment]
@hakkiplaten you can also fork this repository and push a tag if you want to make sure the steps are exactly right. Maintaining the bower.json file in this repo shouldn't be a big issue considering that the version doesn't need to be bumped.

**Comment by ghost** [comment]
> @hakkiplaten you can also fork this repository and push a tag if you want to make sure the steps are exactly right.

@hut8 that would have to be as a last resort.

> Maintaining the bower.json file in this repo shouldn't be a big issue considering that the version doesn't need to be bumped.

Good point!

**Comment by ghost** [comment]
@potench mind taking a last stab at this?
1. Merge https://github.com/nfl/jquery-oembed-all/pull/42
2. Rename this repo's version tag from `0.9RC1` to the standard semantic version format (http://semver.org/) `0.9.0-rc1` (https://github.com/nfl/jquery-oembed-all/issues/43) to satisfy both NPM and Bower 

This way, Rails developers can use this repo directly and no longer have to be dependent on wrappers such as https://github.com/ecleel/jquery-oembed-rails whose last commit is from 2014.

**Comment by joshjordan** [comment]
@potench any word here, friend? Over at https://github.com/tenex/rails-assets we're trying to push semantic version tags to help other tools (like Bower, which rails-assets uses) integrate with libraries like this one.

I bet @hakkiplaten would help you out if you need it!

**Pull request merged**